backend/package.json

```
@@ -9,10 +9,10 @@
   },
   "scripts": {
     "build": "run-p build:*",
-    "build:esbuild": "node build-assets.js",
-    "dev": "node build-assets.js --watch",
-    "check:eslint": "npx eslint --ignore-pattern ../.gitignore static/js",
-    "fix:eslint": "npx eslint --fix --ignore-pattern ../.gitignore static/js",
+    "build:esbuild": "node build-assets.mjs",
+    "dev": "node build-assets.mjs --watch",
+    "check:eslint": "npx eslint",
+    "fix:eslint": "npx eslint --fix",
```

I'm not sure why they were like this, but we don't need `npx` to run these commands as npm scripts. 

```suggestion
    "check:eslint": "eslint",
    "fix:eslint": "eslint --fix",
```

---

Oh, fantastic! Went ahead and axed the `npx`'s for the other scripts as well.